packages/graphql-yoga/src/server.ts

```diff
@@ -98,124 +99,77 @@ export type YogaServerOptions<
   TServerContext extends Record<string, any>,
   TUserContext extends Record<string, any>,
   TRootValue,
-> = {
```

Comment:
Seems like you reformatted this file, prettier should take care of that instead.